rainier-core/src/main/scala/com/stripe/rainier/core/Continuous.scala

```diff
@@ -146,26 +146,22 @@ object Exponential {
   */
 object Beta {
   def apply(a: Real, b: Real): Continuous = new Continuous {
-    def realLogDensity(p: Real): Real =
-      If(p < 0,
-         Real.negInfinity,
-         If(p > 1,
-            Real.negInfinity,
-            (a - 1) * p.log + (b - 1) * (1 - p).log - Combinatrics.beta(a, b)))
+    def realLogDensity(real: Real): Real =
+      If(real < 0, Real.negInfinity, If(real > 1, Real.negInfinity, betaDensity(real)))
```

Comment:
I think this could just be `Uniform.standard.realLogDensity(real) + betaDensity(real)`.